### ISSUE TITLE
Markdown inclusion of images broken after 39db9f48

### ISSUE THREAD

**Issue by eseiler** [doxygen/doxygen#8132]
**Describe the bug**
When including an image file inside markdown, a warning is issued and the image is not included.

```console
/run/shm/example/example.md:1: warning: image file img/cc0_badge.svg\ilinebr is not found in IMAGE_PATH: assuming external image.
```

**Expected behavior**
Prior to the mentioned commit, this worked as expected.

**To Reproduce**
[example.tar.gz](https://github.com/doxygen/doxygen/files/5439375/example.tar.gz)
The example includes `example.md` which in turn tries to include `img/cc0_badge.svg`. The path `img/` is added to `IMAGE_PATH`.
The example page with the image is shown under `md_example.html`, reachable from the `index.html` via the tab `Related Pages` and then clicking on `example`.

**Version**
I ran `git bisect` and the first faulty commit is 39db9f48e6376b62f28c9b85320dd62f76547765
```console
$ doxygen -v
1.9.0 (39db9f48e6376b62f28c9b85320dd62f76547765)
```

**Comment by albert-github** [comment]
Nice corner case with the empty description `[]`, looks like the `[ ]` solves the issue as a workaround (so inserting a space).

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #8130 

Note: I've been thinking about prohibiting an empty string as title because when the image is not reachable and we have an empty title nothing is shown and with a proper title is shown the user knows what the problem is.  I left the empty title as a responsibility of the user.

**Comment by albert-github** [comment]
Code has been integrated in master on github (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
